### ISSUE TITLE
Include container IP and MAC addresses in the output

### ISSUE THREAD

**Issue by maximih** [maximih/cniguru#2]
Current cniguru version only includes the MAC address of the veth pair side residing on the physical node. 

It would be helpful to include the IP and MAC addresses of all interfaces inside the containers (e.g. the container side of a veth pair) which can be retrieved from the linux netns:

```
[root@kh1 ~]# ip netns exec c249290d4af6 ip addr show
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
3: eth0@if126: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default 
    link/ether 0a:58:0a:f4:00:07 brd ff:ff:ff:ff:ff:ff link-netnsid 0
    inet 10.244.0.7/24 scope global eth0
       valid_lft forever preferred_lft forever
```

**Comment by maximih** [comment]
Example of container with multiple interfaces:

```
[root@kh1 ~]# ip netns exec aa41e96576b ip addr show
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
3: eth0@if545: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1460 qdisc noqueue state UP group default 
    link/ether 0a:58:0a:f4:00:d8 brd ff:ff:ff:ff:ff:ff link-netnsid 0
    inet 10.244.0.216/24 scope global eth0
       valid_lft forever preferred_lft forever
5: net0@if546: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default 
    link/ether 0a:58:15:17:5f:01 brd ff:ff:ff:ff:ff:ff link-netnsid 0
    inet 21.23.95.1/25 scope global net0
       valid_lft forever preferred_lft forever
7: net1@if547: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default 
    link/ether 0a:58:15:17:60:01 brd ff:ff:ff:ff:ff:ff link-netnsid 0
    inet 21.23.96.1/25 scope global net1
       valid_lft forever preferred_lft forever
9: net2@if548: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default 
    link/ether 0a:58:15:17:61:01 brd ff:ff:ff:ff:ff:ff link-netnsid 0
    inet 21.23.97.1/25 scope global net2
       valid_lft forever preferred_lft forever

```

**Comment by maximih** [comment]
Fixed in 29f019c2dd6fa5d962605a6e657ce959a442f9e4
